blocks/serialization.py

```diff
@@ -215,10 +218,13 @@ def secure_dump(object_, path, dump_function=dump, **kwargs):
 
     """
     try:
+        logger.debug("Dump object to a temporaty file")
```

Comment:
I'd say "Dumping" as a description of what's being done. "Dump" sounds like a command.